### ISSUE TITLE
Missing export() calls error

### ISSUE THREAD

**Issue by debedb** [klahnakoski/mo-sql-parsing#240]
I am unsure what to make of this error so please bear with me

This is the trace:

```
missing expected call export(mo_sql_parsing.utils, unary_ops)
Exception in thread Thread-3 (worker):
Traceback (most recent call last):
  File "/usr/lib/python3.12/threading.py", line 1073, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.12/threading.py", line 1010, in run
    self._target(*self._args, **self._kwargs)
  File "/app/venv/lib/python3.12/site-packages/mo_imports/__init__.py", line 197, in worker
    _error("Missing export() calls")
  File "/app/venv/lib/python3.12/site-packages/mo_imports/__init__.py", line 208, in _error
    raise Exception(description) from None
Exception: Missing export() calls

```

**Comment by klahnakoski** [comment]
@debedb You happen to have the command that produced this?  It might happen if you import something other than one of the top-level parse functions.  I will look around a bit.

**Comment by debedb** [comment]
@klahnakoski only `format` was imported. But it's hard to reproduce, I will try.

**Comment by klahnakoski** [comment]
Let me try with format too. In the meantime I found this, but it is expected


```
Python 3.10.9 (tags/v3.10.9:1dd9be6, Dec  6 2022, 20:01:21) [MSC v.1934 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from mo_sql_parsing.utils import Call
>>> missing expected call export(mo_sql_parsing.utils, unary_ops)
Exception in thread Thread-3 (worker):
Traceback (most recent call last):
  File "c:\Python310\lib\threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "c:\Python310\lib\threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "c:\Python310\lib\site-packages\mo_imports\__init__.py", line 197, in worker
    _error("Missing export() calls")
  File "c:\Python310\lib\site-packages\mo_imports\__init__.py", line 208, in _error
    raise Exception(description) from None
Exception: Missing export() calls
```

**Comment by debedb** [comment]
but I'm only importing 

```

from mo_sql_parsing import format
```

**Comment by klahnakoski** [comment]
thank you.  I am now investigating a fix.

**Comment by klahnakoski** [comment]
https://github.com/klahnakoski/mo-sql-parsing/commit/0a08be2f717d5f99cfa33da7d7f8668618cae4a8

working on deploy now...

**Comment by debedb** [comment]
Thanks! Genuinely curious as to what is the logic there, since I don't know the internals.

**Comment by klahnakoski** [comment]
When I split large modules into logical files, and they inevitably have cyclic import dependencies.  They are fixed being clear on the order in which modules get imported, and this order is best coordinated by the imports found in the __init__.py file. I try my best to ensure later-imported modules set the correct module-level variables in modules imported earlier, but I often fail because the order changes and I forget which module should set what variable and in what order.  These problems show up when I run "organize imports", which changes import order, or I renamed a module, or I added a module that has its own import order, etc.

The mo-imports is a library I wrote to hunt down bugs I write.  I can use `expect()` to declare a module-level variable is expected to be set from elsewhere.  I use `export()` to actually set those module level variables.  In this case utils.py expects `unary_ops`

https://github.com/klahnakoski/mo-sql-parsing/blob/9523cd49796bf19dfc412f7b3d2817ec8e877c3b/mo_sql_parsing/utils.py#L22

and types.py exports that value

https://github.com/klahnakoski/mo-sql-parsing/blob/9523cd49796bf19dfc412f7b3d2817ec8e877c3b/mo_sql_parsing/types.py#L282

The solution was to ensure `mo_sql_parsing.sql_parser` was imported before the formatter so that the correct import order is maintained.

While writing this, I found out the `unary_ops` is not even needed by `utils.py`.  So it is now removed, and everything I said is irrelevant.
